src/com/projectkorra/projectkorra/PKListener.java

```diff
@@ -1415,12 +1415,9 @@ public void onPlayerSlotChange(PlayerItemHeldEvent event) {
 		BendingPlayer bPlayer = BendingPlayer.getBendingPlayer(player);
 		int slot = event.getNewSlot() + 1;
 		
-		if (bPlayer.getAbilities().get(slot) != null ) {

```

Comment:
This null check is needed, however, it should be checking bPlayer.getAbilities() not the actual slot itself.